pkg/api/handlers/compat/events.go

```diff
@@ -15,77 +16,132 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// filtersFromRequests extracts the "filters" parameter from the specified
+// http.Request.  The paramater can either be a `map[string][]string` as done
```

Comment:
nit of a nit, only change if you've other changes inbound.
```suggestion
// http.Request.  The parameter can either be a `map[string][]string` as done
```